Source/WTF/wtf/SortedArrayMap.h

```diff
@@ -230,51 +227,51 @@ template<typename ArrayType> template<typename KeyArgument> inline bool SortedAr
     return iterator != std::end(m_array) && *iterator == *parsedKey;
 }
 
-constexpr int strcmpConstExpr(const char* a, const char* b)
+constexpr int compareSpansConstExpr(std::span<const char> a, std::span<const char> b)
```

Comment:
I would call this `strcmpSpans`. strcmp has the unique behavior that it truncates the comparison. `equalSpans`, on the other hand, compares not equal when they're not the same size.

Reply:
I don't agree. The word "compare"l whether shortened to "cmp" or not, means a 3 way comparison that can take length into account by sorting the lack of bytes before any specific length values. I don't think the "str" is important to that semantic.